docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -17,63 +25,55 @@ Distributing collections involves three major steps:
 
 .. _config_distribution_server:
 
-Configuring your distribution server or servers
-================================================
+Initial configuration of your distribution server or servers
+============================================================
+
```

Comment:
This isn't really required if you are just using community galaxy as it is pre-configured.  Might be useful calling that out since we are expanding the docs.

Reply:
The server is pre-configured, but you still need to get an API token for Galaxy, right?

Reply:
Changed in 2b84951fc61422551f159ccf5df686beff5fb23e.